app/controllers/messages_controller.rb

```diff
@@ -1,31 +1,29 @@
 class MessagesController < ApplicationController
-  before_action :set_message, only: %i[ show edit update destroy ]
+  before_action :set_message, only: %i[show edit update destroy]
 
   # GET /messages or /messages.json
   def index
     @messages = Message.all
   end
 
   # GET /messages/1 or /messages/1.json
-  def show
-  end
+  def show; end
 
   # GET /messages/new
   def new
     @message = Message.new
   end
 
   # GET /messages/1/edit
-  def edit
-  end
+  def edit; end
 
   # POST /messages or /messages.json
   def create
     @message = Message.new(message_params)
 
     respond_to do |format|
       if @message.save
-        format.html { redirect_to message_url(@message), notice: "Message was successfully created." }
+        format.html { redirect_to message_url(@message), notice: 'Message was successfully created.' }
```

Comment:
* Kindly create 5 different greetings as mentioned in the [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md#:~:text=Create%20a%20table%20for%20storing%20your%20messages%20and%20create%205%20different%20greetings)

Reply:
@Davidosky007  WTF YOU ARE TALKING ABOUT , I have a given CRUD method add new greeting into database. You do not see that or I need to give you a manual injection key insert it into database.